metadata-ingestion/src/datahub/configuration/common.py

```diff
@@ -17,7 +18,7 @@ class Config:
         )  # needed to allow cached_property to work. See https://github.com/samuelcolvin/pydantic/issues/1241 for more info.
 
 
-class TransformerSemantics(Enum):
+class TransformerSemantics(ConfigEnum):
     """Describes semantics for aspect changes"""
 
     OVERWRITE = "OVERWRITE"  # Apply changes blindly
```

Comment:
ConfigEnum already implements `_generate_next_value_`.
So, OVERWRITE = auto() is more pydantic.